docs/integration/intellij-idea.md

```diff
@@ -0,0 +1,56 @@
+# Integration with IntelliJ IDEA
+
+Both plugin and plugin-less integrations use Run Configurations
```

Comment:
Put "Run Configurations" in quotes?

Reply:
I’ve tried it but it kinda forces using quotes in all occurrences which pollutes the text visually.